tunnel/tunnel.go

```diff
@@ -52,16 +53,17 @@ func NewServer(user, address, key, sshAgent, cfgPath string) (*Server, error) {
 		port = args[1]
 	}
 
-	c, err := NewSSHConfigFile(cfgPath)
-	if err != nil {
-		if !errors.Is(err, os.ErrNotExist) {
-			return nil, fmt.Errorf("error accessing %s: %v", host, err)
-		}
-	}
-
-	// If ssh config file doesnt exists, create an empty ssh config struct to avoid nil pointer deference
-	if errors.Is(err, os.ErrNotExist) {
+	if cfgPath == "" {
 		c = NewEmptySSHConfigStruct()
+	} else {
+		c, err := NewSSHConfigFile(cfgPath)
```

Comment:
By using `:=`, the code is creating a new `c` variable rather than using the one define outside of the `else` scope. Please use `=`. This is also causing the app to fail when building.

Reply:
`:=` must be used, because err is not defined, so I defined it.

Reply:
You can do something like:

```go
} else {
  var err error

  c, err = NewSSHConfigFile(cfgPath)

...
```

Reply:
I defined `var err error` above. Should I move it here?